### PR TITLE
change default option related to softmax, test=develop

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

**Review by panyx0718 — COMMENTED** [review]
why make this change?

**Comment by jerrywgz** [comment]
> why make this change?

softmax_with_cross_entropy may cause NaN in models and it can be solved by setting use_cudnn to False

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -1734,7 +1734,7 @@ def sequence_softmax(input, use_cudnn=False, name=None):
     return softmax_out
 
 
-def softmax(input, use_cudnn=True, name=None):
+def softmax(input, use_cudnn=False, name=None):
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py:1737]: add some comments to explain the difference between true or false?

Comment (jerrywgz) [reply python/paddle/fluid/layers/nn.py:1737]: Done

**Review by panyx0718 — APPROVED** [review]

**Pull request merged**